index.html

```diff
@@ -467,21 +448,218 @@ <h4 class="mt-2 underline">91Springboard</h4>
               </div>
           </div>
       </section> -->
-        <div class="py-8 font-work-sans" id="team">
-            <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
-                Our Team
-            </h2>
+    <div class="py-8 font-work-sans" id="team">
+      <h2 class="font-work-sans font-semibold text-4xl text-center">
+        Our Team
+      </h2>
+      <div class="w-full">
+        <h2 class="font-work-sans font-medium text-2xl text-center pt-4">
+          New Delhi
+        </h2>
+        <div class="flex flex-row flex-wrap">
+          <div class="h-full w-full md:w-1/3 pb-4 px-5">
+            <div class="flex flex-col p-5 text-center rounded-md">
+              <img
+                src="./resources/assets/bangalore.png"
+                alt=""
+                srcset=""
+                class="h-40 w-40 rounded-full mx-auto bg-blue-600"
+              />
+              <h1 class="font-work-sans text-2xl py-5">Ekas Preet SIngh</h1>
+              <span class="flex justify-center items-center">
+                <a href="">
```

Comment:
This is not accessible. We should add aria label, and span tag to tell the users that this is Linkdin link.